### ISSUE TITLE
Error when update data content

### ISSUE THREAD

**Issue by eshaudigital** [Datatrics/magento2-integration#13]
**Describe the bug**
I just updated the extension from 1.3.0 tp 1.3.1 and when I run "datatrics:content:update" give error message :

/magento2$ php bin/magento datatrics:content:update --store-id=1
Warning: Invalid argument supplied for foreach() in /data/web/magento2/vendor/datatrics/magento2-integration/Model/ProductData/Repository.php on line 243

**Used versions**
- *Magento Version number 2.4.0*:
- *Open source*:
- *Module version number 1.3.1*:

**To Reproduce**
just run datatrics:content:update 

**Expected behavior**
update data content

**Actual behavior**
error message

**Comment by evs-xsarus** [comment]
Full trace:

```
{"0":"Warning: Invalid argument supplied for foreach() in \/home\/johndoe\/site\/vendor\/datatrics\/magento2-integration\/Model\/ProductData\/Repository.php on line 243","1":"#1 Datatrics\\Connect\\Model\\ProductData\\Repository->addCategoryData() called at [vendor\/datatrics\/magento2-integration\/Model\/ProductData\/Repository.php:122]
#2 Datatrics\\Connect\\Model\\ProductData\\Repository->getProductData() called at [vendor\/datatrics\/magento2-integration\/Model\/Command\/ContentUpdate.php:228]
#3 Datatrics\\Connect\\Model\\Command\\ContentUpdate->prepareData() called at [vendor\/datatrics\/magento2-integration\/Controller\/Adminhtml\/Content\/Update.php:98]
#4 Datatrics\\Connect\\Controller\\Adminhtml\\Content\\Update->execute() called at [vendor\/magento\/framework\/App\/Action\/Action.php:108]
#5 Magento\\Framework\\App\\Action\\Action->dispatch() called at [vendor\/magento\/module-backend\/App\/AbstractAction.php:231]
#6 Magento\\Backend\\App\\AbstractAction->dispatch() called at [vendor\/magento\/framework\/Interception\/Interceptor.php:58]
#7 Datatrics\\Connect\\Controller\\Adminhtml\\Content\\Update\\Interceptor->___callParent() called at [vendor\/magento\/framework\/Interception\/Interceptor.php:138]
#8 Datatrics\\Connect\\Controller\\Adminhtml\\Content\\Update\\Interceptor->Magento\\Framework\\Interception\\{closure}() called at [vendor\/magento\/module-backend\/App\/Action\/Plugin\/Authentication.php:143]
#9 Magento\\Backend\\App\\Action\\Plugin\\Authentication->aroundDispatch() called at [vendor\/magento\/framework\/Interception\/Interceptor.php:135]
#10 Datatrics\\Connect\\Controller\\Adminhtml\\Content\\Update\\Interceptor->Magento\\Framework\\Interception\\{closure}() called at [vendor\/magento\/framework\/Interception\/Interceptor.php:153]
#11 Datatrics\\Connect\\Controller\\Adminhtml\\Content\\Update\\Interceptor->___callPlugins() called at [generated\/code\/Datatrics\/Connect\/Controller\/Adminhtml\/Content\/Update\/Interceptor.php:26]
#12 Datatrics\\Connect\\Controller\\Adminhtml\\Content\\Update\\Interceptor->dispatch() called at [vendor\/magento\/framework\/App\/FrontController.php:159]
#13 Magento\\Framework\\App\\FrontController->processRequest() called at [vendor\/magento\/framework\/App\/FrontController.php:98]
#14 Magento\\Framework\\App\\FrontController->dispatch() called at [vendor\/magento\/framework\/Interception\/Interceptor.php:58]
#15 Magento\\Framework\\App\\FrontController\\Interceptor->___callParent() called at [vendor\/magento\/framework\/Interception\/Interceptor.php:138]
#16 Magento\\Framework\\App\\FrontController\\Interceptor->Magento\\Framework\\Interception\\{closure}() called at [vendor\/magento\/framework\/Interception\/Interceptor.php:153]
#17 Magento\\Framework\\App\\FrontController\\Interceptor->___callPlugins() called at [generated\/code\/Magento\/Framework\/App\/FrontController\/Interceptor.php:26]
#18 Magento\\Framework\\App\\FrontController\\Interceptor->dispatch() called at [vendor\/magento\/framework\/App\/Http.php:116]
#19 Magento\\Framework\\App\\Http->launch() called at [vendor\/magento\/framework\/App\/Bootstrap.php:261]
#20 Magento\\Framework\\App\\Bootstrap->run() called at [pub\/index.php:118]
","url":"\/admin_dlk\/datatrics\/content\/update\/store_id\/1\/key\/6d2789f0c8ffefa800c1466c13cd99c9e30796de0fa43bf0042b945db1897875\/","script_name":"\/index.php","report_id":"5b3913f47c37778821bcf35704a1da9d9376a1e40331c4e783a0f0fe038eecaf"}
```

**Comment by Frank-Magmodules** [comment]
HI @evs-xsarus ,

Thank you for opening this issue, as we can't reproduce this issue, please contact us through our [contact form](https://www.magmodules.eu/support-form.html) with more information about your environment so we debug and fix this for you.

Thanks!

**Comment by evs-xsarus** [comment]
@Frank-Magmodules I'm running "magento/product-enterprise-edition": "2.3.4"

**Comment by Frank-Magmodules** [comment]
HI @evs-xsarus , 

We are happy to share that we've just released the new  [1.3.2](https://github.com/Datatrics/magento2-integration/releases/tag/v1.3.2) version with a fix for this issue. 
Thank you again for the detailed issue report. We are closing this issue now but please feel free to reopen the issue if this still occurs.
